Jenkinsfile

```diff
@@ -17,10 +17,14 @@ node('local') {
             }
         }
 
-        stage('Report tests') {
+        stage('Report Tests') {
             junit '*/target/surefire-reports/*.xml'
         }
 
+        stage('Check Documentation') {
+            sh 'mkdocs build'
```

Comment:
indeed the flag --strict should be present, but this mkdocs isn't consistent - if there's a wrong *.md file written in mkdocs.yml then it's an error, but if there's a mistake in one of the *.md files then it's a warning